.github/workflows/lifecycle.yml

```diff
@@ -12,3 +12,48 @@ on:
 jobs:
   lifecycle:
     uses: kumahq/.github/.github/workflows/wfc_lifecycle.yml@main
+  refresh-active-releases:
+    if: github.event_name == "schedule" || github.event_name == "workflow_dispatch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate GitHub app token
+        id: github-app-token
+        uses: tibdex/github-app-token@021a2405c7f990db57f5eae5397423dcc554159c # v1.7.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: master
+        env:
+          GITHUB_TOKEN: ${{ steps.github-app-token.outputs.token }}
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "~1.18.9"
+      - name: refresh-active-branches
+        run: |
+          REPO="${{ github.repository }}"
+          EARLIEST_RELEASE=$(gh api /repos/${REPO}/releases --jq '[.[] | select(.name | endswith(".0")) | select(.published_at | fromdateiso8601 > (now - 3600 * 24 * 365)) | .name] | last')
```

Comment:
Maybe add a comment here that this is the 1 year policy so it's easier to find if this ever changes?